com.spotify.Client.json

```diff
@@ -136,7 +106,8 @@
                 "install spotify-bin /app/bin/spotify",
                 "install -Dm644 com.spotify.Client.appdata.xml /app/share/appdata/com.spotify.Client.appdata.xml",
                 "cp /usr/bin/ar /app/bin",
-                "ARCH_TRIPLE=$(gcc --print-multiarch) && cp /usr/lib/${ARCH_TRIPLE}/libbfd-*.so /app/lib"
+                "ARCH_TRIPLE=$(gcc --print-multiarch) && cp /usr/lib/${ARCH_TRIPLE}/libbfd-*.so /app/lib",
+                "ln -s /usr/lib/x86_64-linux-gnu/libcurl.so.4.5.0 /app/lib/libcurl-gnutls.so.4"
```

Comment:
We do ship a 32bit build, not sure if it even needs the same version of curl since its old though.

Reply:
Oh you already commented on that. The `ARCH_TRIPLE=$(gcc --print-multiarch)` usage above should do.